### PR TITLE
Added 'none' board type for simple powerup/powerdown commands

### PR DESCRIPTION
**Pull request by jessehamner**
Some light editing of a few descriptions, plus glue code to enable the 'NoBoard' Target type.
Added some new attributes for each pin, given the potential need for pins to be set to input or output.

### CODE REVIEW

File: test/integration/jenkinsnode/cistack/none.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import os
```
Comment (plauche) [inline test/integration/jenkinsnode/cistack/none.py]: I'd suggest we pick a name which isn't so close to the Python keyword `None`.

File: test/integration/jenkinsnode/ci.py [hunk 2]
```diff
@@ -47,6 +47,10 @@ def main():
     logging.debug("Script started %s." % str(NOW))
     logging.info("Arguments have been parsed.")
 
+#    sofile=open('/tmp/stdoutfile.txt', 'w')
```
Comment (plauche) [inline test/integration/jenkinsnode/ci.py]: Should this code by commented/removed or triggered by a flag?

File: test/integration/jenkinsnode/ci.py [hunk 3]
```diff
@@ -66,6 +70,31 @@ def main():
...
+
+# set up the GPIO pins as requested
+    target.setupboard()
+    sleep(1)
+
+# create a dict of Pin objects:
+    pins = target.getpins()
+    logging.debug(str(pins))
+
+    if args.powerup is True:
```
Comment (plauche) [inline test/integration/jenkinsnode/ci.py]: What is the difference between the `args.powerup` flag and `args.command` with a value of `"powerup"`? They appear to be different flags with identical functionality.

Comment (jessehamner) [inline test/integration/jenkinsnode/ci.py]: Old habits die hard, and I always forget exact keywords. I'll pull out the 'command' functions.

File: test/integration/jenkinsnode/ci.py (hunk 2 above)
Comment (jessehamner) [inline test/integration/jenkinsnode/ci.py]: Ah, that should be removed. Thanks for catching it.

File: test/integration/jenkinsnode/cistack/none.py (hunk 1 above)
Comment (jessehamner) [inline test/integration/jenkinsnode/cistack/none.py]: Very fair. I thought of that but wanted to throw the PR up quickly -- the iOBC is benefiting from this functionality and I need to hurry up.

**Pull request closed without merging**